kolibri/plugins/device_management/assets/src/views/select-content-page/index.vue

```diff
@@ -166,9 +166,20 @@
         // This could produced unexpected behavior for users.
         this.showUpdateProgressBar = true;
         return this.downloadChannelMetadata().then(() => {
+          this.updateChannelTotalResouce();
           this.showUpdateProgressBar = false;
         });
       },
+      updateChannelTotalResouce() {
+        let availableChannel = this.availableChannels;
+        for (let i = 0; i < availableChannel.length; i++) {
+          if (this.channel.id === availableChannel[i].id) {
+            // Update channel total resources.
+            this.channel.total_resources = availableChannel[i].total_resources;
```

Comment:
`this.channel` is vuex state and cannot be modified like this. Note that `downloadChannelMetadata` is an vuex action; you must make any changes to vuex state from actions.

Take a look at the vuex 1.0 docs here: https://github.com/vuejs/vuex/tree/1.0/docs/en

Reply:
It's done @indirectlylit, Thanks to your directions.